core/src/main/java/org/elasticsearch/index/query/TermsQueryBuilder.java

```diff
@@ -185,43 +192,108 @@ public String fieldName() {
     }
 
     public List<Object> values() {
```

Comment:
I think it needs to remain public since it is part of the public API of this class?